phones/migrations/0017_relaynumber_enabled.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.14 on 2022-07-27 21:24
+
+from django.db import migrations, models
```

Comment:
I'm suggesting a larger change in PR #2261, to ensure that there is a database-level default for `enabled`. It probably is not a large issue for this change, since there are little or no phones being created in production, but I want us to get better about adding columns while the stakes are low.

Reply:
praise: Thanks for doing that! I'll include it and now we can refer back to it for future columns.